pkg/providers/vmware/mapper/mapper.go

```diff
@@ -298,7 +299,11 @@ func (r *VmwareMapper) MapDisk(vmSpec *kubevirtv1.VirtualMachine, dv cdiv1.DataV
 	}
 
 	vmSpec.Spec.Template.Spec.Volumes = append(vmSpec.Spec.Template.Spec.Volumes, volume)
-	vmSpec.Spec.Template.Spec.Domain.Devices.Disks = append(vmSpec.Spec.Template.Spec.Domain.Devices.Disks, kubevirtDisk)
+	disks := append(vmSpec.Spec.Template.Spec.Domain.Devices.Disks, kubevirtDisk)
+	sort.Slice(disks, func(i, j int) bool {
```

Comment:
To ensure the disks are attached in the same order as the devices on the source VM. Since the the import controller is iterating over a map of DVs, MapDisk gets called in a nondeterministic order which can result in the disks being in random order.

Reply:
:+1: